src/qmflows/fileFunctions.py

```diff
@@ -3,11 +3,11 @@
 __all__ = ['yaml2Settings']
 
 import yaml
-from typing import Union
+from typing import AnyStr
 from .settings import Settings
 
 
-def yaml2Settings(xs: Union[bytes, str]) -> Settings:
```

Comment:
Are those two types equivalent? https://docs.python.org/3/library/typing.html#typing.AnyStr

Reply:
Yup, it's an alias